.travis.yml

```diff
@@ -1,28 +1,44 @@
-language: php
+language: bash
+dist: xenial
 
-php:
-  - 7.1
+services:
+-   docker
 
 cache:
-  directories:
-    - ~/.composer/cache
-    - ~/.npm
+    directories:
+    -   $CACHE_DIR
 
+addons:
+    apt:
+        update: true
 env:
-  - ELASTIC_SEARCH_INDEX_PREFIX=""
+    global:
+    -   CACHE_DIR=$HOME/.cache/docker
+    -   CACHE_FILE_IMAGE=$CACHE_DIR/php-fpm.tar.gz
+    -   DOCKER_COMPOSE_VERSION=1.24.0
+    -   TRAVIS_USER_ID=$(id -u)
+    -   TRAVIS_GROUP_ID=$(id -g)
 
 before_install:
-  - phpenv config-add project-base/docker/php-fpm/php-ini-overrides.ini
-  - echo '' | pecl install redis-4.1.1
-  - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - phpenv config-rm xdebug.ini || return 0
-  - composer global require hirak/prestissimo
+-   sudo rm /usr/local/bin/docker-compose
+-   sudo add-apt-repository ppa:rmescandon/yq -y
```

Comment:
https://docs.travis-ci.com/user/installing-dependencies/#adding-apt-sources

Reply:
If it could be used like addons I would love to use it, but YQ is not in the addons packages and I find adding them using sourcelines more dirty than this.

Reply:
OK then